### PR TITLE
Address CSRCs encryption without header extensions

### LINKED ISSUE #14: Address CSRCs without header extensions
@murillo128 sayeth: 

> CSRCs can only be encrypted if the x bit is set to one, as the "defined by profile" field must be present in the RTP packet to allow the receiver to differentiate it from a {{RFC3711}} encrypted packet. In case that the RTP packet does not contain any header extension, {{RFC2550}} allows to include a 0 length header extension.

### PR DESCRIPTION
**Pull request by murillo128**
Fixes #14

### CODE REVIEW

File: draft-uberti-avtcore-cryptex.md [hunk 1]
```diff
@@ -262,6 +262,13 @@ required by {{RFC8285}}.
 Specifically, the encrypted portion MUST include any CSRC identifiers, any
 RTP header extension (except for the first 4 bytes), and the RTP payload.
 
+CSRCs identifiers can only be encrypted if the x bit is set to one, as the
```
Comment (juberti) [inline draft-uberti-avtcore-cryptex.md:265]: This should go into the RTP Header Processing section, and mention that an empty header MUST be included in this case (and said empty headers must be tolerated during receiving).

Comment (murillo128) [reply draft-uberti-avtcore-cryptex.md:265]: I agree, doing the change now

**Comment by juberti** [comment]
Made some updates to be more specific, PTAL and LMK what you think

**Comment by murillo128** [comment]
it is much better now

**Pull request merged**